pkg/cmd/objects/import/import.go

```diff
@@ -16,16 +17,13 @@ import (
 )
 
 type ImportOptions struct {
-	Config *config.Config
-	IO     *iostreams.IOStreams
-
-	SearchClient func() (*search.Client, error)
-
-	Index string
-
+	Config                         *config.Config
+	IO                             *iostreams.IOStreams
+	SearchClient                   func() (*search.Client, error)
+	Index                          string
 	AutoGenerateObjectIDIfNotExist bool
-
-	Scanner *bufio.Scanner
+	Scanner                        *bufio.Scanner
+	BatchSize                      int
```

Comment:
Those were kept separated for readability, could you please add the line breaks back, please? 🙏

Reply:
that's is done 🙏